src-plugins/medSegmentation/msegAlgorithmPaintToolbox.cpp

```diff
@@ -556,6 +559,47 @@ void AlgorithmPaintToolbox::activateStroke()
     emit installEventFilterRequest(m_viewFilter);
     addBrushSize_shortcut->setEnabled(true);
     reduceBrushSize_shortcut->setEnabled(true);
+
+    activateCustomedCursor(); // cursor for painting
+}
+
+void AlgorithmPaintToolbox::activateCustomedCursor()
+{
+    if (!currentView) // no data
+    {
+        return;
+    }
+
+    // Get size of the brush
+    medAbstractImageView * imageView = dynamic_cast<medAbstractImageView *>(currentView);
+    int si = (int)((float)(m_brushSizeSpinBox->value()*2.0f)/imageView->sliceThickness());
+
+    // Create shape of the new cursor
+    QPixmap *pix = new QPixmap(si,si);
```

Comment:
Leak? I don't see why this needs to be heap allocated - we pass by copy when creating the QCursor on 595, and the only other use of this is as an argument to a stack-variable (painter).